end-to-end-test/remote/specs/core/plots.screenshot.spec.js

```diff
@@ -449,6 +449,12 @@ describe('plots tab screenshot tests', function() {
         browser.click('.coloringLogScale');
         waitForAndCheckPlotsTab();
     });
+    it('plots tab with structural variant coloring', () => {
+        goToUrlAndSetLocalStorage(
```

Comment:
@adamabeshouse make sure to download screenshot for this before merging to master